### ISSUE TITLE
Stores registered with feathers-pinia not showing-up in the devtools

### ISSUE THREAD

**Issue by miguelrk** [marshallswain/feathers-pinia#16]
**TLDR:** Original `defineStore()` called with `options.storeId` instead of `options.id` and pinia requires `options.id` to register store in vue devtools.

### Issue

Stores created with feathers-pinia not showing-up in vue devtools (at least when using vue2).

### Context

From pinia docs: 
> This name, also referred as id, is necessary and is used by Pinia to connect the store to the devtools.

### Solution

In this case, the only fix should be to rename the expected `options.storeId` to `options.id` to at least have a more transparent/direct mapping to the pinia API.  This change does not break tests and remains backwards compatible.

So basically change the following code:
```diff
  // Create and initialize the Pinia store.
  const storeOptions: any = makeServiceStore({
-   storeId: options.id || `service.${options.servicePath}`,
+   id: options.id || `service.${options.servicePath}`,
    idField: options.idField || idField || 'id',
    clientAlias,
    servicePath,
    clients,
    Model: options.Model,
    state,
    getters,
    actions,
    whitelist,
  })
  function useStore(pinia?: Pinia): any {
    const useStoreDefinition = definePiniaStore(storeOptions)
    ...
  }
```

**Comment by miguelrk** [comment]
### Additional Comments

Currently, the feathers-pinia wrapper for `defineStore()` calls the original `defineStore()` method (here `definePiniaStore()` to avoid naming conflict) with a single argument and passing the store's id in storeOptions.storeId:
```ts
import { defineStore as definePiniaStore } from 'pinia'

const useStoreDefinition = definePiniaStore(storeOptions)
```
The original `defineStore()` of pinia apparently accepts either 1, 2, or 3 arguments with the following (simplified) signatures:
```ts
defineStore(id, options): Store // the default, more explicit approach (as seen in the pinia docs)

defineStore(options): Store // expects options.id, not options.storeId (what feathers-pinia currently passes)

defineStore(id, storeSetup, options): Store // store setup is a function
```

The `defineStore()` of feathers-pinia currently only accepts one argument, the `options` (so it matches #2 from above):
```ts
defineStore(options): Store // expects options.id, not options.storeId (what feathers-pinia currently passes)
```

I would say supporting all 3 options like pinia does is overkill, however if the `defineStore()` of feathers-pinia is resulting more and more distinct from the original `defineStore()` of pinia, I would suggest **maybe renaming the method in feathers-pinia to avoid confusion and keep it only accepting 1 argument, namely the options.**

**Comment by marshallswain** [comment]
We should support these two:

```
defineStore(id, options): Store 

defineStore(options): Store
```

Your current PR resolves the primary use case of the second one.  In the case of the 2nd, we can make `options.id` optional and use the servicePath if `options.id` isn't provided.

**Comment by marshallswain** [comment]
I'm going to keep this open until we support 

```
defineStore(id, options): Store 
```

**Comment by miguelrk** [comment]
Perfect! Alright I'll try cooking something up to support both 👍🏼
